# linux port and AX-12A baud rate
usb_port: "/dev/ttyUSB0"
dxl_baud_rate: 1000000

# control table addresses
addr_torque_enable: 24
addr_goal_position: 30
addr_present_position: 36
addr_speed: 32
addr_cw_compliance_slope: 28
addr_ccw_compliance_slope: 29

# AX-12A min max
dxl_minimum_position_value: 0
dxl_maximum_position_value: 1023

# Protocol version
protocol_version: 1.0

# Set the dynamixel id for various joints
joint_pan:
  ID: 1
  zero_config: 512
  speed: 200
  cw_compliance_slope: 32
  ccw_compliance_slope: 32
  min: 0
  max: 1023
joint_tilt:
  ID: 2
  zero_config: 819
  speed: 200
  cw_compliance_slope: 144
  ccw_compliance_slope: 144
  min: 200
  max: 820
joint_elbow:
  ID: 3
  zero_config: 819
  speed: 200
  cw_compliance_slope: 64
  ccw_compliance_slope: 64
  min: 300
  max: 900
joint_paddle:
  ID: 4
  zero_config: 512
  speed: 200
  cw_compliance_slope: 32
  ccw_compliance_slope: 32
  min: 0
  max: 1023